src/main/scala/org/constellation/http/Loadbalancer.scala

```diff
@@ -62,6 +82,23 @@ class Loadbalancer(port: Int = 9000, host: String = "localhost") {
         .flatMap(size => Ok(s"$size"))
   }
 
+  private def maintenance(routes: HttpRoutes[IO]): HttpRoutes[IO] = Kleisli { req: Request[IO] =>
+    for {
+      shouldReturnServiceUnavailable <- OptionT.liftF(shouldRedirectToMaintenance)
+      res <- if (shouldReturnServiceUnavailable) {
+        val retryAfterTime = HttpDate.unsafeFromInstant(Instant.now.plus(30, ChronoUnit.MINUTES))
```

Comment:
I think that usually rollback takes not more than 5min, so setting it 10 would be enough. If LB will be disabled at the time, it will return another retry anyway

Reply:
EOD it can be stored in app conf 🤔